packages/discovery-provider/src/queries/get_comments.py

```diff
@@ -141,22 +144,32 @@ def get_track_comments(args, track_id, current_user_id=None):
             .outerjoin(
                 CommentReaction, Comment.comment_id == CommentReaction.comment_id
             )
+            .outerjoin(
+                MutedUser,
+                and_(
+                    MutedUser.muted_user_id == Comment.user_id,
+                    MutedUser.user_id == current_user_id,
+                ),
+            )
             .group_by(Comment.comment_id)
             .filter(
                 Comment.entity_id == track_id,
                 Comment.entity_type == "Track",
+                Comment.is_delete == False,
```

Comment:
side note: I intentionally removed this but I need to re-do it 🙈
Planning to work on that today